src/patterns.rs

```diff
@@ -16,6 +16,7 @@ const BMP: &'static [u8] = b"BM";
 const BGP: &'static [u8] = b"BPG\xfb";
 const RGB: &'static [u8] = b"\x01\xda";
 const FLIF: &'static [u8] = b"FLIF";
+const ICO: &'static [u8] = b"\x00\x00\x01\x00";
```

Comment:
Are `.CUR` images still exists, btw? Have not seen them for a decades.

Reply:
I haven't seen them either. Also, it [isn't listed](https://github.com/image-rs/image#21-supported-image-formats) as one of [image](https://github.com/image-rs/image)'s supported image types.